src/Tizen.NUI/src/internal/Common/ProcessorController.cs

```diff
@@ -0,0 +1,203 @@
+
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        private delegate void LayoutProcessorEventCallbackType();
+
+        private ProcessorCallback callback = null;
+
+        public event EventHandler ProcessorOnceEvent
+        {
+            add
+            {
+                if (processorOnceEventHandler == null)
+                {
+                    processorOnceEventCallback = OnProcessorOnceEvent;
+                }
+
+                processorOnceEventHandler += value;
+            }
+
+            remove
+            {
+                processorOnceEventHandler -= value;
+
+                if (processorOnceEventHandler == null)
+                {
+                    processorOnceEventCallback = null;
+                }
+            }
+        }
+
+        public event EventHandler ProcessorEvent
+        {
+            add
+            {
+                if (processorEventHandler == null)
+                {
+                    processorEventCallback = OnProcessorEvent;
+                }
+
+                processorEventHandler += value;
+            }
+
+            remove
+            {
+                processorEventHandler -= value;
+
+                if (processorEventHandler == null)
+                {
+                    processorEventCallback = null;
+                }
+            }
+        }
+
+        public event EventHandler LayoutProcessorEvent
+        {
+            add
+            {
+                if (layoutProcessorEventHandler == null)
+                {
+                    layoutProcessorEventCallback = OnLayoutProcessorEvent;
+                }
+
+                layoutProcessorEventHandler += value;
+            }
+
+            remove
+            {
+                layoutProcessorEventHandler -= value;
+
+                if (layoutProcessorEventHandler == null)
+                {
+                    layoutProcessorEventCallback = null;
+                }
+            }
+        }
+
+        private ProcessorController() : this(Interop.ProcessorController.New(), true)
+        {
+        }
+
+        internal ProcessorController(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+            callback = new ProcessorCallback(Process);
+            Interop.ProcessorController.SetCallback(SwigCPtr, callback);
+        }
+
+        private static ProcessorController instance = null;
+
+        public static ProcessorController Instance
+        {
+            get
+            {
+                if(instance == null)
+                {
+                    instance = new ProcessorController();
+                }
+                return instance;
+            }
+        }
+
+        public void Process()
+        {
+            processorOnceEventHandler?.Invoke(this, null);
+            processorOnceEventHandler = null;
+            processorEventHandler?.Invoke(this, null);
+            layoutProcessorEventHandler?.Invoke(this, null);
```

Comment:
we can call
ProcessorOnceEvent?.Invoke(this, null);
ProcessorOnceEvent = null;
ProcessorEvent?.Invoke(this, null);
LayoutProcessorEvent?.Invoke(this, null);

directly.